<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPdc</class>
 <widget class="QMainWindow" name="MPdc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MPdc</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 255, 200);</string>
   </property>
   <widget class="QLabel" name="LblShowPic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>cam1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnTakePicture">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnCallPolice">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>报警</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblDateTime">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LblSafeOrNot">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>336</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Abyssinica SIL</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>安全</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
